src/spinner/src/Spinner.js

```diff
@@ -74,6 +92,22 @@ class Spinner extends PureComponent {
       </Box>
     )
   }
+
+  componentDidMount() {
```

Comment:
I don't think that's something we should worry about, as realistically it won't happen. If there's some unusual case where developer needs to update delay after spinner is already mounted, they should use a `key` on `<Spinner>`, so that a new one is created.